dvc/command/experiments.py

```diff
@@ -474,6 +476,18 @@ def show_experiments(
     if kwargs.get("only_changed", False):
         td.drop_duplicates("cols")
 
+    extra_render_args = {}
+    if html:
+        td.dropna("rows")
+        td.column("Experiment")[:] = [
+            # remove tree characters
+            str(x).encode("ascii", "ignore").strip().decode()
+            for x in td.column("Experiment")
+        ]
+        rel = kwargs.get("out") or "dvc_plots"
```

Comment:
Better to use a variable for this `dvc_plots`? Keep it synchronize with what in `dvc/command/plots.py`

Reply:
Makes sense. I doubt I had is whether to use the same default folder as plots o to have a different one instead (i.e. dvc_exp?)

Wdyt?

cc @dberenbaum

Reply:
@karajan1001 Where this kind of constants should be placed in order to be accessed by different commands (i.e. `dvc plots` / `dvc exp`) ? 

A constant under `Repo` like the existing `Repo.DVC_DIR`?

Reply:
I'm not quite sure, for myself I would place them to `dvc.plot.Plot`?

Reply:
@pared thoughts?

Reply:
Tracked in https://github.com/iterative/dvc/issues/7088